packages/uploadthing/src/client.ts

```diff
@@ -258,12 +259,66 @@ export const DANGEROUS__uploadFiles = async <TRouter extends FileRouter>(
       fileUrl: genUrl,
       url: genUrl,
     };
+
+    if (!fileData) {
+      console.log("[UT] Did not receive file data from UT");
+    } else {
+      console.log("[UT] Trying to simulate the webhook");
+
+      await conditionalDevServer(fileData.fileData, presigned.key);
+    }
+
     return ret;
   });
 
   return Promise.all(fileUploadPromises);
 };
 
+const conditionalDevServer = async (file: FileData, fileKey: string) => {
+  if (process.env.NODE_ENV !== "development") return;
```

Comment:
this won't work on Vite apps. We should be careful using the `process` object on client side things

Reply:
For real. Well, I guess I can reuse the approach you have for Astro, like the one with try-catch